examples/basic_usage/README.md

```diff
@@ -12,13 +12,12 @@ Note that this example may create resources which cost money. Run
 
 | Name | Version |
 |------|---------|
-| terraform | ~> 0.14.0 |
+| terraform | ~> 1.0 |
+| aws | ~> 3.38 |
 
 ## Providers ##
 
-| Name | Version |
-|------|---------|
-| aws | ~> 3.38 |
+No providers.
```

Comment:
Shouldn't the default provider be listed here?

Reply:
Not according to `terraform-docs`.  I'm not exactly sure why, but that is how it does it now; the `aws` line has moved from the `Providers` section to the `Requirements` section.

Reply:
I believe what's happened is that because of the deletion of the empty `aws` provider it no longer sees any providers defined for the configuration that aren't used by modules. It is likely https://github.com/terraform-docs/terraform-docs/issues/530 and we just haven't refreshed this README. I get the same output using `v0.14.1`.

Reply:
It doesn't seem to happen everywhere, though.  See [here](https://github.com/cisagov/distributed-subnets-tf-module/blob/lineage/skeleton/examples/basic_usage/README.md), for instance.

This isn't a huge issue, but the inconsistency perturbs me.

Reply:
Thanks for that link, @mcdonnnj!

Reply:
> It doesn't seem to happen everywhere, though. See [here](https://github.com/cisagov/distributed-subnets-tf-module/blob/lineage/skeleton/examples/basic_usage/README.md), for instance.
> 
> This isn't a huge issue, but the inconsistency perturbs me.

I agree, but I think we are all on board with using the output of `terraform-docs`, rather than trying to amend it manually, right?

Reply:
It's because [that configuration uses the provider in something other than a module](https://github.com/cisagov/distributed-subnets-tf-module/blob/b2c953cc2d0eae023eccefc1d75648cad24a8477/examples/basic_usage/main.tf#L13-L16). In this configuration the provider is *only* used in a module and that's why it's handled differently.

Reply:
Thanks @mcdonnnj - that makes sense!

Reply:
Right, @mcdonnnj's link cleared things up for me.  That's why I resolved the conversation.

No changes necessary.  Full speed ahead! :motor_boat: